plugins/package-managers/node/src/main/kotlin/Yarn2.kt

```diff
@@ -119,6 +128,31 @@ class Yarn2(
          * The amount of package details to query at once with `yarn npm info`.
          */
         private const val BULK_DETAILS_SIZE = 1000
+
+        /**
+         * The name of the manifest file used by Yarn 2+.
+         */
+        private const val MANIFEST = "package.json"
```

Comment:
And one more thing: While the trend has not been started with this PR, we should not continue adding `private` constants to a `companion object`. Using a `companion object` makes sense to scope `public` constants only.

IMO these should become `private` top-level constants (some applies to the function), but this can be addressed separately.